src/cmd/manage_nsfs.js

```diff
@@ -431,24 +541,36 @@ async function manage_account_operations(action, data, config_root, config_root_
 
 
 /**
- * list_config_file will list all the config files (json) in a given path
- * @param {fs.PathLike} full_path
+ * list_config_file will list all the config files (json) in a given config directory
+ * @param {string} config_path
  */
-async function list_config_file(full_path) {
-    const entries = await fs.promises.readdir(full_path);
+async function list_config_file(config_path) {
```

Comment:
I think this we can change the function name:
```suggestion
async function list_config_files(config_path) {
```